enterprise_access/apps/api/v1/tests/test_subsidy_access_policy_views.py

```diff
@@ -2785,7 +2785,7 @@ def test_can_redeem_assigned_policy(
         # Only an errored assignment exists.
         {'has_cancelled_assignment': False, 'has_failed_assignment': True},
         # No assignment exists for the learner/content pair to check.
-        {'has_cancelled_assignment': False, 'has_failed_assignment': True},
+        {'has_cancelled_assignment': False, 'has_failed_assignment': False},
```

Comment:
This is not related to my changes, just noticed that we had a duplicate test case.